app/assets/javascripts/bastion/components/notification.service.js

```diff
@@ -16,8 +16,15 @@ angular.module('Bastion.components').service("Notification", ['$interpolate', 'f
         return result;
     }
 
-    this.setSuccessMessage = function (message, context) {
-        foreman.toastNotifications.notify({message: interpolateIfNeeded(message, context), type: 'success'});
+    this.setSuccessMessage = function (message, options) {
+        /* eslint-disable no-unused-expressions, angular/ng_definedundefined*/
+        var baseOptions = {};
+        var fullOptions = {};
+        (options === undefined) && (options = {});
```

Comment:
Since this is the only line we want to disable the `no-unused-expressions` rule on, you can move the eslint disable line above it and then re-enable the rule after it.